cypress/pageobject/Users/UserCreation.ts

```diff
@@ -48,20 +125,6 @@ export class UserCreationPage {
       .click();
   }
 
-  selectHomeFacility(name: string) {
-    this.clickElementById("home_facility");
-    this.selectOptionContainingText(name);
-  }
-
-  setInputDate(dateElementId: string, dateValue: string) {
-    cy.clickAndTypeDate("#" + dateElementId, dateValue);
-  }
-
-  selectDropdownOption(dropdownId: string, optionText: string) {
-    this.clickElementById(dropdownId);
-    this.selectOptionContainingText(optionText);
-  }
-
   verifyElementContainsText(elementId: string, expectedText: string) {
```

Comment:
- remove this type of usage and replace it with the reusable function cy.verifyContentPresence()

Reply:
@nihal467 These  function is called from managepatients.cy.ts so i kept it